app/grandchallenge/evaluation/tasks.py

```diff
@@ -218,6 +222,12 @@ def retry_with_delay():
 
     evaluation.update_status(status=Evaluation.EXECUTING_PREREQUISITES)
 
+    civ_sets = [
```

Comment:
This might be a strange optimisation, but this increases the speed at which the `cache.lock` releases and hence indirectly the job-creation throughput overall.